en/docs/get-started/quick-start-guide.md

```diff
@@ -1,34 +1,41 @@
+Welcome to the WSO2 Micro Integrator Quick Start Guide, your step-by-step tutorial for getting started with one of the most powerful integration tools. WSO2 MI enables you to build, deploy, and manage integration solutions with ease, providing flexibility and scalability to connect applications, services, and systems.
+
+In this guide, you'll learn the basics of setting up and using WSO2 MI to create and deploy a basic integration flow with minimal hassle.
 
 ## Prerequisites
 
 The following software and configurations are required to proceed with this tutorial:
 
-- **Visual Studio Code (VS Code):** with the [Micro Integrator for VS Code](https://marketplace.visualstudio.com/items?itemName=WSO2.micro-integrator) extension installed.
-- **Java Development Kit (JDK):** Version 11 or 17 is required. Ensure the JDK is properly configured in your system's PATH environment variable.
+- **Java Development Kit (JDK):** Version 11 or 17 is required. Set up a [JDK that is compatible]({{base_path}}/install-and-setup/setup/reference/product-compatibility/#tested-jdks). Ensure the JDK is properly configured in your system's PATH environment variable.
 
     !!! Info
         For more information on setting the `JAVA_HOME` environment variable for different operating systems, see the [Install and Setup]({{base_path}}/install-and-setup/install/installing-mi) documentation.
 
-- **Apache Maven:** Ensure Apache Maven is installed and its path is correctly set within the system's PATH environment variable.
+  - **Apache Maven:** Ensure [Apache Maven](https://maven.apache.org/download.cgi) is installed (version 3.6.0 onwards) and its path is correctly set within the system's PATH environment variable.
+
+    !!! info
+        For more information on installing Apache Maven, see the [Apache Maven documentation](https://maven.apache.org/install.html).
+
+- **A Rest Client:** Download and install any REST Client of your choice such as [Postman](https://www.postman.com/) or [cURL](https://curl.se/). We will be using Postman in this tutorial. 
 - **WSO2 Micro Integrator 4.3.0 Runtime:** Set up WSO2 Micro Integrator 4.3.0 runtime on your machine.
     1. Download the Micro Integrator 4.3.0 distribution as a ZIP file from [here](https://github.com/wso2/micro-integrator/releases/download/v4.3.0/wso2mi-4.3.0.zip).
     2. Extract the ZIP file. Hereafter, this extracted folder will be referred to as the `<MI_HOME>` folder.
+- **Visual Studio Code (VS Code):** with the [Micro Integrator for VS Code](https://marketplace.visualstudio.com/items?itemName=WSO2.micro-integrator) extension installed.
 
 After completing the steps above, follow the instructions below to set up the workspace:
 
+## Step 1 - Configure MI server
+
+We need to configure the downloaded and extracted WSO2 MI server in the Micro Integrator extension installed VS Code to run the integration solution. Let's proceed with the following steps.
```

Comment:
Let's use `you` instead of `we` in the documentation. There are several other places as well.